### ISSUE TITLE
(6-1.5.2) Nether Portal Loop

### ISSUE THREAD

**Issue by GroeneAppel** [AllTheMods/ATM-6#1144]
**Describe the bug**
Two players are encountering a bug on my server. When they enter a Nether Portal they are constantly teleported back and forth and report experiencing extremely high latency. They remain trapped until they disconnect. After which they will find themselves either at the original overworld coordinates in the Nether, or at the Nether coordinates in the Overworld. 

Both players affected are not EU based (server location) and playing on a EU server. EU players (myself included) do not experience this issue at all.
I suspect latency plays a role for this bug to occur. 

**To Reproduce**
1. Have a high (100+?) ping. 
2. Build and light a Nether Portal
3. Enter the portal
4. End up in a loop
5. Disconnect
6. End up at the Nether coords in the Overworld or vica versa. 

**Additional context**
Seems to be similar to:
#783

**Comment by The-DataDoctor** [comment]
I had the same issue when using the portal charm (the one that lets you pass portals without wait time). Putting the charm out of the inventory helped.

**Comment by GroeneAppel** [comment]
> I had the same issue when using the portal charm (the one that lets you pass portals without wait time). Putting the charm out of the inventory helped.

In this case neither players had a portal charm, walking through the Nether Portal with no gear triggers the bug for them.

**Comment by Pdiddy973** [comment]
Update to 1.5.6, see if issue persist

**Comment by Kipex** [comment]
This has been reported by others about the NetherPortalFix mod, which seems to cause more problems in multiplayer than it fixes. We also had everyone getting stuck in a loop between the Overworld and the Nether until you end up disconnecting or suffocating to death. We experienced this in 1.5.1 and 1.5.3 and as the mod hasn't been updated at any point, we have kept NetherPortalFix disabled through all versions, which fixed the issue.
